qcodes/dataset/sqlite_base.py

```diff
@@ -382,8 +410,109 @@ def perform_db_upgrade_1_to_2(conn: SomeConnection) -> None:
     else:
         raise RuntimeError(f"found {n_run_tables} runs tables expected 1")
 
-    log.info('Succesfully upgraded database version 1 -> 2.')
-    set_user_version(conn, 2)
+
+@upgrader
+def perform_db_upgrade_2_to_3(conn: SomeConnection) -> None:
+    """
+    Perform the upgrade from version 2 to version 3
```

Comment:
does it make sense to list in the docstring what the upgrade consists of? or do we have this info somewhere else? or do we just look at the code?:)

Reply:
That's indeed a fine place to put it. Done.